tests/core/filtering/test_existing_filter_instance.py

```diff
@@ -1,18 +1,14 @@
 import pytest
 
+import pytest_asyncio
+
 from web3._utils.threads import (
     Timeout,
 )
-from web3.providers.eth_tester import (
-    EthereumTesterProvider,
-)
 
 
 @pytest.fixture()
 def filter_id(w3):
-    if not isinstance(w3.provider, EthereumTesterProvider):
```

Comment:
Do you know why this was here in the first place? Does git blame give any insight? 🤔

Reply:
I looked at the history of that line and couldn't tell.  Were there other providers used for running tests pre-2018?

Reply:
I'm not sure. Looks like it is running fine now though 🤷‍♀️